etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -141,6 +142,16 @@ func (s *simplePrinter) MemberList(resp v3.MemberListResponse) {
 	}
 }
 
+func (s *simplePrinter) EndpointHealth(hs []epHealth) {
+	for _, h := range hs {
+		if h.Error == "" {
+			fmt.Fprintf(os.Stderr, "%s is healthy: successfully committed proposal: took = %v\n", h.Ep, h.Took)
+		} else {
+			fmt.Fprintf(os.Stderr, "%s is unhealthy: failed to commit proposal: %v", h.Ep, h.Error)
```

Comment:
@gyuho 
When I am trying the failed cases, I noticed there might be an "\n" missing at the end of this message. Can you confirm? If so, I will also need to fix this in master branch and backport the fix.

Reply:
Originally the error messages were sent to `errc` and then printed in separate lines. Here, all errors are printed in one line.

Reply:
Yes, good catch. Can we fix?

Reply:
I'll send out the fix and changelog for these cherry picks later today.